clang/utils/TableGen/ClangDiagnosticsEmitter.cpp

```diff
@@ -970,10 +942,11 @@ struct DiagTextPrinter : DiagTextVisitor<DiagTextPrinter> {
   void VisitPlural(PluralPiece *P) {
     Result += "%plural{";
     assert(P->Options.size() == P->OptionPrefixes.size());
-    for (unsigned I = 0, End = P->Options.size(); I < End; ++I) {
-      if (P->OptionPrefixes[I])
-        Visit(P->OptionPrefixes[I]);
-      Visit(P->Options[I]);
+    for (const auto &[Prefix, Option] :
```

Comment:
This captures the `zippy` object tuple by reference.  Is this valuable?

Reply:
WDYM by valuable? It's value is writing less code to index into different arrays or spelling out .first/.second below.

Reply:
I see. Yeah, then the additional & is not useful. Will remove it

Reply:
Done.